pages/women_page.rb

```diff
@@ -0,0 +1,179 @@
+class WomenPage
+    PAGE_TITLE = 'WOMEN'
+
+    def page_title
+        #TBD to move to parent page
+        browser.find_element(class: 'cat-name')
+    end
+
+#TITLES OF CATEGORIES
+    def categories_filter
+        browser.find_element(xpath: "//span[contains(@class,'layered_subtitle') and text()='Categories']")
```

Comment:
You seem to already have updated version with moving locating element to separate method

Reply:
Done